nameable.rb

```diff
@@ -0,0 +1,7 @@
+class Nameable
+  def initialize; end
```

Comment:
- [ ] **[OPTIONAL]** This `initialize` method is unused and so unnecessary. I would recommend you remove it